src/types/types.ts

```diff
@@ -208,7 +209,8 @@ export type CallState = {
     screen_sharing_session_id: string;
     owner_id: string;
     host_id: string;
-    recording?: CallRecordingState;
+    recording?: CallJobState;
+    live_captions?: CallJobState;
```

Comment:
So we are using nouns here :p

Reply:
@streamer45 Hmm... Yah, looks like it -- I followed what we were doing https://github.com/mattermost/mattermost-plugin-calls/blob/4920d771d4783d084ae4b07ed171f2536b9c2eeb/server/state.go#L85-L86
`recording`, `transcription`, `live_captions`.
Up to you if you want to change it, now would be the time.

Reply:
It's fine, I surrender, we can't really win entropy.

Reply:
😂